pkg/controller/render/render_controller.go

```diff
@@ -526,6 +599,19 @@ func (ctrl *Controller) syncGeneratedMachineConfig(pool *mcfgv1.MachineConfigPoo
 	}
 	glog.V(2).Infof("Pool %s: now targeting: %s", pool.Name, pool.Spec.Configuration.Name)
 
+	// TODO(jkyros): Hacked this in, take it out later when/if you figure out where it should go
+	// I initially picked here because I wasn't sure if we'd need to update OsImageURL or
+	// include special files in our render (e.g. machineconfig-to-ignition, etc)
+	if ctrlcommon.IsLayeredPool(pool) {
+
+		glog.Infof("Pool %s is s special pool, rendering config to imagestream", pool.Name)
+		// jkyros: Just put this here for now since it's convenient
+		err = ctrl.experimentalRenderToImageStream(pool, generated)
```

Comment:
Are we cool with the "push to imagestream approach" or do we need to figure out a service endpoint and retrieve this from inside the build?

Reply:
For the purposes of this PR, I'd like to just evaluate "risks to not-layered".

Reply:
(It does add and version bump some dependencies but nothing that seemed overtly dangerous )